Formula/aide.rb

```diff
@@ -30,6 +31,11 @@ class Aide < Formula
   uses_from_macos "curl"
 
   def install
+    # fix `fatal error: 'error.h' file not found`
+    inreplace "include/aide.h", "#include \"error.h\"", ""
```

Comment:
This probably needs to be reported upstream, no?

Reply:
Yeah, I will submit a patch PR.

Reply:
Submitted upstream at https://github.com/aide/aide/pull/85